docker/Dockerfile.static

```diff
@@ -8,6 +8,7 @@ RUN apk add --update \
   bash \
   bcc-dev \
   bcc-static \
+  benchmark \
```

Comment:
Dockerfile.static has a fairly specific purpose of exercising statically linked build configuration. Is there a reason this test-only dep needs to be added?

Reply:
Does it build all targets statically? E.g. `gtest-dev` is also in this list (as well as llvm18-gtest).

Reply:
No, it disables tests: https://github.com/bpftrace/bpftrace/blob/870547c1a4a4974318d9fa2036429328da9fd11f/.github/include/static.sh#L20

The gtest packages might just be vestigial, then

Reply:
Updated.